apis/apps/v1beta4/util.go

```diff
@@ -79,3 +79,15 @@ func (n Names) Data() string {
 func (n Names) BootstrapUser() string {
 	return fmt.Sprintf("%s-%s", n.Object.GetName(), "bootstrap-user")
 }
+
+func ServicePortsToEndpointPorts(servicePorts []corev1.ServicePort) []discoveryv1.EndpointPort {
```

Comment:
Move it to e2e package

Reply:
> Move it to e2e package

Ok, this method has been moved into e2e, I will delete this code,